### PR TITLE
ML 2.8: K_Nearest_Neighbour_Algorithm(A)

### PR DESCRIPTION
**Pull request by sanjana-iyengar**
<hr>

## Description 📜

This issue will help readers in acquiring all the knowledge that one needs to know about the basics of K Nearest Neighbours Algorithm.
Fixes # 2809

<hr>

## Type of change 📝

- [x] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [x] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Classification is a process of arranging a given set of information into each unique class. Order can be executed on the two sorts of information organized just as unstructured or structured. Classes are frequently referred to as Labels or Targets which holds various classes. For instance, classifying different fruits. 
+
+Regression is a problem where our taget holds continuos or real values. Like prediction of age of an individual. 
+
+
+K-Nearest Neighbor classifier is one of the introductory classifiers, which every data science learner should be aware of.  KNN aims for pattern recognition tasks. 
+K-Nearest Neighbor also known as KNN is a supervised learning algorithm that can be used for regression as well as classification problems. Generally, it is used for classification in machine learning. 
+
+[Reference Audio](https://drive.google.com/drive/folders/1OObJrcyS45EDw5QMryfbebOCk9_IlCbA?usp=sharing)
```
Comment (abhisheks008) [inline Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md:13]: @sanjana-iyengar Please add a thumbnail like others have done and then hyperlink the audio file link with that. You can take help from the Demo folder, where we have discussed about how to add your audio or video files.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md (hunk 1 above)
Comment (sanjana-iyengar) [reply Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md:13]: Sure

**Review by Isha307 — CHANGES_REQUESTED** [review]
@sanjana-iyengar The audio file you had attached with the thumbnail is about SVM.
Please look into that.

**Comment by sanjana-iyengar** [comment]
@Isha307 Corrected.

File: Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md [hunk 2]
```diff
@@ -12,5 +12,5 @@ K-Nearest Neighbor also known as KNN is a supervised learning algorithm that can
 
 Click the image for more info
 
-[![ML 2.8 : K Nearest Neighbours (A)](https://user-images.githubusercontent.com/83051558/135641006-9d2b57ff-f9ba-4b57-8647-434352fc6549.png)](https://drive.google.com/file/d/1vTV1FUkQEr_Z1rECvrfXygagfjChnAkK/view?usp=sharing "SVM")
+[![ML 2.8 : K Nearest Neighbours (A)](https://user-images.githubusercontent.com/83051558/135641006-9d2b57ff-f9ba-4b57-8647-434352fc6549.png)](https://drive.google.com/drive/folders/1OObJrcyS45EDw5QMryfbebOCk9_IlCbA?usp=sharing)
```
Comment (Isha307) [inline Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md:15]: Change your file to this:
https://drive.google.com/file/d/1uL45FsHKzBF6NQ5xMHOT20SpojZ1epZ9/view?usp=sharing

**Review by Isha307 — CHANGES_REQUESTED** [review]

File: Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md (hunk 2 above)
Comment (sanjana-iyengar) [reply Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md]: Done

**Review by Isha307 — APPROVED** [review]

**Comment by Isha307** [comment]
@sanjana-iyengar Your PR got merged, Congrats🥳

**Pull request merged**